recipes/hurl/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "4.1.0" %}
+
+package:
+  name: hurl
+  version: {{ version }}
+
+source:
+  url: https://github.com/Orange-OpenSource/hurl/archive/{{ version }}.tar.gz
+  sha256: 3356f64158e6dc5f2e29c37eee80a43332b1734baa7380356affeb5160ffca09
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - cargo-bundle-licenses
+    - pkg-config
+  host:
+    - libxml2
+    - libcurl
+  run:
```

Comment:
These requirements should not be needed if you would use the normal c compiler on Windows.

Reply:
Hi @xhochy thanks for the feedback! I've changes the recipe to only use the c compiler and pushed the new version.

But I've struggled to remove:

```
   host:
     - libxml2
     - libcurl
```
(I've tried to remove the two dependencies but the CI didn't compile on Linux, Mac and Windows).
Hurl is a binary written in Rust and depends on two crates, libxml and curl. These 2 crates provide Rust bindings for the C libraries libxml2.0 and libcurl. As they're needed for build, I initially put them in the `b

Reply:
Libxml and co is fine. I want you to remove the vc runtime and ucrt explicit mentions.

Reply:
Hi @xhochy thanks for your review and for the clarification.
I've modified build.sh and blt.bat to use libxml from conda (removed all vcpkg stuff), and also I've removed the vc runtime and ucrt mentions.